module/zfs/zfs_ioctl.c

```diff
@@ -5255,6 +5256,8 @@ zfs_ioc_pool_reopen(zfs_cmd_t *zc)
 	spa_close(spa, FTAG);
 	return (0);
 }
+#endif
```

Comment:
Did not see matching #if .....

Reply:
the matching if is above (if KERNEL). I have added a comment after this endif to make it more apparent.